DifferentiationInterface/ext/DifferentiationInterfaceMooncakeExt/onearg.jl

```diff
@@ -1,42 +1,47 @@
-struct TapirOneArgPullbackPrep{Y,R} <: PullbackPrep
+struct MooncakeOneArgPullbackPrep{Y,R} <: DI.PullbackPrep
     y_prototype::Y
     rrule::R
 end
 
-function DI.prepare_pullback(f, backend::AutoTapir, x, ty::Tangents)
+function DI.prepare_pullback(f, backend::AutoMooncake, x, ty::DI.Tangents)
     y = f(x)
     rrule = build_rrule(
-        get_tapir_interpreter(),
+        get_interpreter(),
         Tuple{typeof(f),typeof(x)};
-        safety_on=backend.safe_mode,
-        silence_safety_messages=false,
+        debug_mode=false,  # TODO: modify
+        silence_debug_messages=false,
```

Comment:
The intention moving forwards is that both `debug_mode` and `silence_debug_messages` will be specified via `Config`, so I would recommend getting both of these fields from `backend`.

Reply:
If the `backend.config === nothing`, then you should just build a `Mooncake.Config()` object via the default constructor, and pull out the required fields.